.github/workflows/main.yml

```diff
@@ -36,3 +36,5 @@ jobs:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           IS_FORK: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == true }}
           NETLIFY_TEST_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Codecov test coverage
```

Comment:
Also, should be do this just for one node version and OS? Any benefit from uploading all results? Does Codecov knows to differentiate?

Reply:
Yes, it merges them together, and you can even see per-OS results.
That's useful for OS-specific code. For example, if some code only runs on Windows, and another never runs on Windows, we need to run both to get the proper test coverage.